tslint.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Eslint supports TS now, maybe just focus on Eslint?

https://dev.to/robertcoopercode/using-eslint-and-prettier-in-a-typescript-project-53jb

https://javascriptplayground.com/typescript-eslint/

https://eslint.org/blog/2019/01/future-typescript-eslint